.eslintrc

```diff
@@ -32,3 +36,11 @@ rules:
 
   # specify the maximum depth callbacks can be nested (off by default)
   max-nested-callbacks: [1, 2]
+
+  # Forbid the import of external modules that are not declared in the
+  # package.json's dependencies, devDependencies, optionalDependencies or
+  # peerDependencies
+  # (note: we disable this rule because packages inside `src` behave like
+  # aliases: e.g.: require('elements/foo') works).
```

Comment:
I don't think it would solve it. The problem is that we have `./src` and `./` and javascript "roots" and eslint doesn't like that. Alternatively, we can use relative paths for every internal import but that makes it harder to move files around (e.g: move from elements to components).